lib/cocoapods/bazel.rb

```diff
@@ -19,6 +19,10 @@ def self.post_install(installer:)
       UI.titled_section 'Generating Bazel files' do
         workspace = installer.config.installation_root
         sandbox = installer.sandbox
+
+        # Ensure we declare the sandbox (Pods/) as a package so each Pod (as a package) belongs to sandbox root package instead
+        FileUtils.touch(File.join(installer.config.sandbox_root, 'BUILD.bazel'))
```

Comment:
instead of this, can we `build_files[installer.config.sandbox_root.relative_path_from(workspace).to_s]` so it's consistent with the other build files?

Reply:
yup makes sen!

Reply:
hmmm `build_files[installer.config.sandbox_root.relative_path_from(workspace).to_s]` breaks CI test because file comparison fails: https://github.com/bazel-ios/cocoapods-bazel/pull/39/checks?sha=ca7c9b5496604ac82848c519a410d58a1d258199
This approach gives green build... Going to merge the current implementation instead since it does not break test and it is more straightforward imo